erizo_controller/erizoClient/src/webrtc-stacks/FirefoxStack.js

```diff
@@ -1,36 +1,54 @@
 import Logger from '../utils/Logger';
 import BaseStack from './BaseStack';
 
+const possibleLayers = [
```

Comment:
@kekkokk I've changed the previous parameters to have these ones, which is quite similar to what we get from Chrome's Simulcast

Reply:
seems much better

Reply:
who can't reach the resolution, the sender side publishing 640x480 or the receiver subscribing to it?

Reply:
the receiver subscribing it will never receive the full 640x480 res because by default we set 2 layers and we push into the array the scaleresolution by 2 and 3.

Reply:
that's probably because there's no bw limit now, but you can still set a global max bw with `maxVideoBW` parameter when publishing.

Reply:
sorry, I read it wrong, it might be a bug in the way I add it to the parameters.